nautobot/core/testing/filters.py

```diff
@@ -41,7 +41,7 @@ def get_filterset_test_values(self, field_name, queryset=None):
                 # randomly break out of loop after 2 values have been selected
                 if len(test_values) > 1 and random.choice([True, False]):  # noqa: S311  # suspicious-non-cryptographic-random-usage
                     break
-                if value[field_name] and value["count"] < qs_count:
+                if value[field_name] and value["count"] <= qs_count:
```

Comment:
Change made here to account for scenarios when there are exactly two choices.

Reply:
This test should fail when there are exactly two choices. You need more than two choices to accurately test a multiplechoicefilter